DirectProgramming/DPC++FPGA/ReferenceDesigns/anr/README.md

```diff
@@ -17,20 +17,50 @@ This DPC++ reference design demonstrates a highly optimized image sensor adaptiv
 ## Purpose
 This FPGA reference design demonstrates a parameterizable image processing pipeline that implements an Adaptive Noise Reduction (ANR) algorithm using a bilateral filter. See the [Additional Design Information Section](#additional-design-information) for more information on the ANR algorithm itself and how it was implemented for the FPGA.
 
-## License  
+## License
 Code samples are licensed under the MIT license. See
 [License.txt](https://github.com/oneapi-src/oneAPI-samples/blob/master/License.txt) for details.
 
+### Using Visual Studio Code*  (Optional)
+
+You can use Visual Studio Code (VS Code) extensions to set your environment,
+create launch configurations, and browse and download samples.
+
+The basic steps to build and run a sample using VS Code include:
+ - Download a sample using the extension **Code Sample Browser for Intel oneAPI Toolkits**.
+ - Configure the oneAPI environment with the extension **Environment Configurator for Intel oneAPI Toolkits**.
+ - Open a Terminal in VS Code (**Terminal>New Terminal**).
+ - Run the sample in the VS Code terminal using the instructions below.
+ - (Linux only) Debug your GPU application with GDB for Intel® oneAPI toolkits using the Generate Launch Configurations extension.
+
+To learn more about the extensions, see
+[Using Visual Studio Code with Intel® oneAPI Toolkits](https://software.intel.com/content/www/us/en/develop/documentation/using-vs-code-with-intel-oneapi/top.html).
+
+After learning how to use the extensions for Intel oneAPI Toolkits, return to this readme for instructions on how to build and run a sample.
+
 ## Building the Reference Design
 
+> **Note**: If you have not already done so, set up your CLI
```

Comment:
This is the first appearance of the term CLI in this document, it should be defined here.

Reply:
@ishaghosh27 My impression is that CLI is a widely used term among developers and does not need to be defined in a sample README. Do you agree?

Reply:
After talking with Isha and looking at the other samples, including Get Started samples, I don't see a reason to define CLI. 
1. A user will likely have gone through the toolkit or component Get Started (documents on IDZ) and Command Line Interface (CLI) is defined in those documents.
2. I looked a few Get Started samples and did not see it spelled out in those samples. 
I'm not opposed to changing it here, but if I change it here, I have to go through 150 sample READMEs and change it in all of them as well. So let me know if you still feel strongly about spelling it out. Otherwise I will leave it as is.

Reply:
I think we should leave this as is to be consistent with the other readmes

Reply:
I can live with that, thanks for looking into it.